### DISCUSSION TITLE
Show the images in Gallery view at the correct size

### DISCUSSION THREAD

**Discussion by AlyssonNeto** [nocodb/nocodb#3645] [Q&A]
How can I show the images in gallery mode like this example in the airtable, without deforming them.

And would it be possible to show the title also without the title label?

![msedge_3DlcFIKYd9](https://user-images.githubusercontent.com/1557770/190268105-25a79987-840c-4156-b4c2-693babcf9d58.png)
Noco

![msedge_AOZhctk7lA](https://user-images.githubusercontent.com/1557770/190268088-18ed30ea-8fae-4361-9f5a-a768864e7ead.png)
Airtable

Thanks

**Comment by dstala** [comment]
@AlyssonNeto 
one PR related to this is merged as part of https://github.com/nocodb/nocodb/issues/3599
another improvement work for gallery view is work in progress https://github.com/nocodb/nocodb/issues/3647

We will explore if we can map `Primary value` below the image without associated column header.
